geowebcache/nestedstorage/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
```

Comment:
Ah this is the new feature.

Perhaps you can do once change to change to ID_REFERNCE in once pull request, and then do this new feature in a second?